### DISCUSSION TITLE
DAB USB dont work

### DISCUSSION THREAD

**Discussion by francistrabac** [rern/rAudio#1467] [Bug Reports]
Hello,

I can’t get USB DAB to work. I have an error message. See below

mediamtx 1.1.1-1
# rtl_test -t
Found 1 device(s):
  0:  Realtek, RTL2838UHIDIR, SN: 00000001

Using device 0: Generic RTL2832U OEM
Found Rafael Micro R828D tuner
Supported gain values (29): 0.0 0.9 1.4 2.7 3.7 7.7 8.7 12.5 14.4 15.7 16.6 19.7 20.7 22.9 25.4 28.0 29.7 32.8 33.8 36.4 37.2 38.6 40.2 42.1 43.4 43.9 44.5 48.0 49.6 
Sampling at 2048000 S/s.
Testing tuner range. This may take a couple of minutes..

mediamtx.service- Ready-to-use RTSP / RTMP / LL-HLS / WebRTC server and proxy
     Loaded: loaded (/usr/lib/systemd/system/mediamtx.service; enabled; preset: disabled)
     Active: failed (Result: exit-code) since Tue 2023-10-24 19:02:31 CEST; 13min ago
   Duration: 42ms
   Main PID: 1124 (code=exited, status=203/EXEC)
        CPU: 6ms

Oct 24 19:02:31 RuneaudioBUREAU systemd[1]: Started Ready-to-use RTSP / RTMP / LL-HLS / WebRTC server and proxy.
Oct 24 19:02:31 RuneaudioBUREAU (mediamtx)[1124]: mediamtx.service: Failed to execute /usr/bin/mediamtx: Exec format error
Oct 24 19:02:31 RuneaudioBUREAU (mediamtx)[1124]: mediamtx.service: Failed at step EXEC spawning /usr/bin/mediamtx: Exec format error
Oct 24 19:02:31 RuneaudioBUREAU systemd[1]: mediamtx.service: Main process exited, code=exited, status=203/EXEC
Oct 24 19:02:31 RuneaudioBUREAU systemd[1]: mediamtx.service: Failed with result 'exit-code'.

Best regards

**Comment by hjfisch** [comment]
Not working for me too .
After boot it seems to be ok , but when I try to scan the usb-stick is gone.

mediamtx 1.2.0-1
# rtl_test -t
Found 1 device(s):
  0:  Realtek, RTL2838UHIDIR, SN: 00000001

Using device 0: Generic RTL2832U OEM
usb_claim_interface error -6
Failed to open rtlsdr device #0.

mediamtx.service- Ready-to-use RTSP / RTMP / LL-HLS / WebRTC server and proxy
     Loaded: loaded (/usr/lib/systemd/system/mediamtx.service; disabled; preset: disabled)
     Active: inactive (dead)

Oct 31 18:18:37 hjfSound mediamtx[366]: 2023/10/31 18:18:37 INF MediaMTX v1.2.0
Oct 31 18:18:37 hjfSound mediamtx[366]: 2023/10/31 18:18:37 INF configuration loaded from /etc/mediamtx/mediamtx.yml
Oct 31 18:18:37 hjfSound mediamtx[366]: 2023/10/31 18:18:37 INF [RTSP] listener opened on :8554 (TCP), :8000 (UDP/RTP), :8001 (UDP/RTCP)
Oct 31 18:18:37 hjfSound mediamtx[366]: 2023/10/31 18:18:37 INF [RTMP] listener opened on :1935
Oct 31 18:18:37 hjfSound mediamtx[366]: 2023/10/31 18:18:37 INF [HLS] listener opened on :8888
Oct 31 18:18:37 hjfSound mediamtx[366]: 2023/10/31 18:18:37 INF [WebRTC] listener opened on :8889 (HTTP)
Oct 31 18:18:37 hjfSound mediamtx[366]: 2023/10/31 18:18:37 INF [SRT] listener opened on :8890 (UDP)
Oct 31 20:05:39 hjfSound systemd[1]: Stopping Ready-to-use RTSP / RTMP / LL-HLS / WebRTC server and proxy...
Oct 31 20:05:39 hjfSound systemd[1]: mediamtx.service: Deactivated successfully.
Oct 31 20:05:39 hjfSound systemd[1]: Stopped Ready-to-use RTSP / RTMP / LL-HLS / WebRTC server and proxy.
